test-data/divmod.yaml

```diff
@@ -392,41 +393,55 @@ post:
 messages:
   - "0: Possible division by zero (r2 != 0)"
 ---
+test-case: positive modulo positive range
+
+pre: ["r1.type=number", "r1.value=6", "r2.type=number", "r2.value=[1, 5]"]
+
+code:
+  <start>: |
+    r1 %= r2
+
+post:
+  - r1.type=number
+  - r1.value=[0, 4]
+  - r2.type=number
+  - r2.value=[1, 5]
+---
 test-case: negative modulo positive
 
 pre: ["r1.type=number", "r1.value=-13"]
 
 code:
   <start>: |
-    r1 %= 3
+    r1 %= 4
```

Comment:
Why the change?

Reply:
because -13 % 3 is 0 which wouldn't necessarily distinguish the case from other paths that set the answer to 0 so I wanted a non-zero result case